### DISCUSSION TITLE
Activate comitup Hotsport after reconnecting to WLAN

### DISCUSSION THREAD

**Discussion by cgebken** [outdoorbits/little-backup-box#170] [Q&A] [answered]
Hi all,
I just installed the newest LBB Version and chose to also install comitup. After rebooting I was able to connect to the lbb wlan. 
After browsing to http://10.41.0.1 I was prompted to enter the credentials for reconnecting to the "normal" wifi-network - I did that. However, now the LBB is connected to said network and it won't create the hotspot anymore. Even after installing comitup via SSH again and rebooting, it still won't create the hotspot. 
Can you give me a hint on how to reenable the hotspot and/or comitup?

**Comment by outdoorbits** [comment, marked as answer]
Hi, to open the access to LBB convenient, comitup disables it's webUI as soon as the connection is established.
The easiest way to get the comitup webUI back would be to boot where the known WiFi is not available. Maybe you just switch off your WiFi and boot LBB.
But why do you want to do that? If you need the IP you have two options: Use a display or setup email notification. So you'll get a mail containing the links to LBB.

**Reply by cgebken** [reply]
Great, thanks! Simply didn't realise that the hotspot will only be created when no known WiFi is available.

**Reply by mrsparkle331** [reply]
> Hi, to open the access to LBB convenient, comitup disables it's webUI as soon as the connection is established. The easiest way to get the comitup webUI back would be to boot where the known WiFi is not available. Maybe you just switch off your WiFi and boot LBB. But why do you want to do that? If you need the IP you have two options: Use a display or setup email notification. So you'll get a mail containing the links to LBB.

I think it would be useful to be able to reactivate comitup webui even when connected to a wifi network. Sometimes I don't have the ability to control the wifi at the location if its at a friends house or at a hotel. 

Also it would be useful in testing at home before travelling. Turning off the SSID at home would disconnect everything else in the house.

**Comment by outdoorbits** [comment]
That would certainly be very desirable. However, comitup deactivates its web interface itself after a connection has been established with a wifi. LBB then takes over the server address for its own interface, but has nothing to do with the deactivation of the comitup interface.
I don't know of any way to keep the commitup interface active, but I'm grateful for a solution.

**Reply by mrsparkle331** [reply]
hmm okay thanks for the info. I'm digging into the comitup documentation and seeing if there is a way to do what I'm thinking of. I'll put up a PR if there is something I figure out.

**Reply by qdmt** [reply]
When I ran into this, I simply started over, but this time avoided entering the wifi creds. Then two key things I needed to do:
1) ignore the captive portal (that shows list of networks)
2) enter http**S**://yourlbbip.

#2 applies only after all setup is done and comitup has been installed.

**Comment by outdoorbits** [comment]
Of course, this maybe is the real outdoor-mode. But if you would like to use mail-notifications or even cloud-storage options, you need to connect to your wifi.

**Comment by rkosick** [comment]
Not sure if this is helpful for you, but when I want to disconnect lbb from a known network to get the comitup webUI back again, I connect to lbb via SSH on the known WIFI and IP address and execute the following command:

comitup-cli d

This resets the known connection and comitup webUI will be available again with the standard AP.

**Comment by outdoorbits** [comment]
Good idea, I will implement this into the config.

**Comment by outdoorbits** [comment]
It's implemented. Please have a look at settings.
